### DISCUSSION TITLE
Blank Dev App

### DISCUSSION THREAD

**Discussion by susansiow** [quasarframework/quasar#9756] [CLI - Capacitor / Cordova modes] [answered]
Today, I persistently run into a blank app with this error - **Uncaught TypeError: Cannot read property 'signals' of undefined** , though I've updated caniuse-lite with this script > **npx browserslist@latest --update-db** several times.

Kindly assist, thanks!
<img width="899" alt="screenshot-2" src="https://user-images.githubusercontent.com/53169547/122863896-37198680-d356-11eb-915f-96077b613e3f.png">
<img width="400" alt="screenshot" src="https://user-images.githubusercontent.com/53169547/122863908-3b45a400-d356-11eb-8f43-a2f256462d7b.png">

**Comment by metalsadman** [comment, marked as answer]
browser compatibility is set up in your package.json, see https://quasar.dev/quasar-cli/browser-compatibility#configuring-compatibility.

**Reply by susansiow** [reply]
Hi @metalsadman ,

The blank page is caused by browser compatibility?

I've tried to replaced the browser list to one value - "defaults" but it doesn't fix the issue, kindly assist further!

<img width="228" alt="screenshot" src="https://user-images.githubusercontent.com/53169547/122872108-88c80e00-d362-11eb-91a9-24686ec7d02c.png">

**Comment by susansiow** [comment]
After spending several hours of debugging, I've discovered that the cause is not related to browserslist. It's in fact related to an import script error in a Vuex file, what a day!

Nevertheless, below warning persists each time when I run **quasar dev** command (though I've followed the warning instruction and run this - npx browserslist@latest --update-db). Is there a fix on this or should I ignore it?

```
Browserslist: caniuse-lite is outdated. Please run:
npx browserslist@latest --update-db
```

**Reply by hawkeye64** [reply]
@susansiow Personally, I just ignore. I ran the update command once and it messed up the build. I had to remove node_modules and re-install.

As for the first issue, you can always run `quasar build --debug` as that will give you better info on your error (missing end tag, typo, etc).

**Reply by susansiow** [reply]
Hi @hawkeye64 

Noted, thank you for your kind advice!
